File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.cpp
class Solution {
public:
    bool uniqueOccurrences(vector<int>& arr) {
        unordered_map<int, int> counter;
        for(auto x:arr){
            counter[x]++;
        }
        
        vector<int> x;
        for(auto [key, value]: counter){
            for(int j=0; j<x.size(); j++){
                if(x[j]==value){
                    return false;
                }
            }
            x.push_back(value);
        }
        return true;
    }
};

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py
class Solution:
    def uniqueOccurrences(self, arr: List[int]) -> bool:
        count = Counter(arr)
        print(count)
        x = list()
        
        for key, val in count.items():
            if val in x:
                return False
            x.append(val)
        return True